http/swagger.yml

```diff
@@ -8954,19 +8959,54 @@ components:
               type: boolean
             level:
               $ref: "#/components/schemas/CheckStatusLevel"
-    Threshold:
+    ThresholdBase:
       properties:
         level:
          $ref: "#/components/schemas/CheckStatusLevel"
         allValues:
```

Comment:
From our discussion earlier, sounds like this field doesn't exist anymore.

Reply:
I haven't got to those changes yet.